DB/projects.json

```
@@ -2505,31 +3231,64 @@
         "link": "https://github.com/vaibhav-xt/Certificate-Generator",
         "title": "Certificate Generator",
         "description": "This is a certificate generator application that take inputs and generate certificate also we cand do download. ",
-        "tech": ["html", "css", "javascript", "html2canva"]
+        "tech": [
+          "html",
+          "css",
+          "javascript",
+          "html2canva"
+        ]
       },
       {
-        "link": "https://vaibhav-maurya.netlify.app/",
+        "link": "https://github.com/vaibhav-xt/Portfolio",
         "title": "Portfolio site",
         "description": "A portfolio project showcases an individual's or a professional's work, skills, and accomplishments in a concise and visually appealing manner to demonstrate their expertise and attract potential opportunities.",
-        "tech": ["css", "javascript", "html"]
+        "tech": [
+          "css",
+          "javascript",
+          "html"
+        ]
       },
       {
         "link": "https://github.com/vaibhav-xt/Music-Player",
         "title": "Music Player",
         "description": "Music Player",
-        "tech": ["css", "javascript", "html"]
+        "tech": [
+          "css",
+          "javascript",
+          "html"
+        ]
       },
       {
         "link": "https://github.com/vaibhav-xt/Vanilla-Javascript-Projects/tree/master/Color%20Flipper",
         "title": "Color Flipper",
         "description": "Color Flipper is a vanilla javascript project.",
-        "tech": ["css", "javascript", "html"]
+        "tech": [
+          "css",
+          "javascript",
+          "html"
+        ]
       },
       {
         "link": "https://github.com/vaibhav-xt/Vanilla-Javascript-Projects/tree/master/Reviews",
         "title": "Testimonials Carousal",
         "description": "This is the testimonial carousal made with vanilla javascript. ",
-        "tech": ["css", "javascript", "html"]
+        "tech": [
+          "css",
+          "javascript",
+          "html"
+        ]
+      },
+      {
+        "link": "https://github.com/vaibhav-xt/OIBSIP_Pizza-delivery-app",
+        "title": "Pizza Delivery Web App",
+        "description": "Full Stack Pizza Delivery Application",
+        "tech": [
+          "React",
+          "Redux",
+          "MongoDB",
+          "Node Js",
+          "Express JS"
```

convert all to lowercase 
```suggestion
          "react",
          "redux",
          "mongodb",
          "nodejs",
          "expressjs"
```